acceptance/config/beaker/jenkins/el-6-x86_64-mdca.cfg

```diff
@@ -8,6 +8,13 @@ HOSTS:
     platform: el-6-x86_64
     template: Delivery/Quality Assurance/Templates/vCloud/centos-6-x86_64
     hypervisor: vcloud
+    puppetbin: /opt/puppetlabs/agent/bin/puppet
+    puppetbindir: /opt/puppetlabs/agent/bin
+    puppetvardir: /opt/puppetlabs/agent/cache
+    puppetpath: /etc/puppetlabs/agent
+    facterbindir: /opt/puppetlabs/agent/bin
+    hierabindir: /opt/puppetlabs/agent/bin
+
```

Comment:
The best place for these variables to be set would be https://github.com/puppetlabs/puppet-server/blob/master/acceptance/config/beaker/options.rb

This A) reduces the need for duplication of these settings in other beaker config files and B) ensures that these values are available for CI consumption.

The reason for B) is that the CI jobs dynamically generate Beaker host.cfg using the `genconfig2` utility provided by `sqa-utils`.